src/redismodule.h

```diff
@@ -60,10 +60,13 @@ typedef long long ustime_t;
 #define REDISMODULE_OPEN_KEY_NOEXPIRE (1<<19)
 /* Avoid any effects from fetching the key */
 #define REDISMODULE_OPEN_KEY_NOEFFECTS (1<<20)
+/* Allow access expired key that haven't deleted yet */
+#define REDISMODULE_OPEN_KEY_ACCESS_EXPIRED (1<<21)
```

Comment:
Just verify, this also implies `REDISMODULE_OPEN_KEY_NOEXPIRE`. right?

Reply:
Another question, I see that this new flag will effect subkey (hash fields) as well. But `REDISMODULE_OPEN_KEY_NOEXPIRE` does not effect subkey (correct me if I am wrong)?
Isn't it an inconsistency in the API?

Reply:
I discussed with @MeirShpilraien offline. For the protocol, we already agreed in the past to avoid any lazy delete of subkeys that might in turn lead to unexpected deletion of an open key. That is, won't be any subkey or key deletion and in turn won't be any "effect".